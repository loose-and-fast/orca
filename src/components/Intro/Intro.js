import React from 'react';
import { Grid } from 'semantic-ui-react';
import PropTypes from 'prop-types';
import './Intro.css'

const Intro = () => (
  <Grid.Row centered columns={2}>
    <img src="orca-logo-white.png" alt="Not working today" className="logo" />
  </Grid.Row>
);

Intro.propTypes = {};

export default Intro;
