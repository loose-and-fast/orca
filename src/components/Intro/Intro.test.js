import React from 'react'

import Intro from './Intro'

describe('Intro', () => {
  let component, props

  beforeEach(() => {
    props = {}
    component = shallow(<Intro {...props} />)
  })

  it('should', () => {
    expect(component).toMatchSnapshot()
  })
})