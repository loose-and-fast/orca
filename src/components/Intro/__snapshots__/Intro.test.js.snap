// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro should 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Intro />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "centered": true,
      "children": <img
        alt="Not working today"
        className="logo"
        src="orca-logo-white.png"
/>,
      "columns": 2,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "alt": "Not working today",
        "className": "logo",
        "src": "orca-logo-white.png",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "centered": true,
        "children": <img
          alt="Not working today"
          className="logo"
          src="orca-logo-white.png"
/>,
        "columns": 2,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Not working today",
          "className": "logo",
          "src": "orca-logo-white.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
